atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -512,7 +511,15 @@ def select_and_start_cluster(self, platform):
                     ctx.try_again_later(self.failure_retry_delay)
                     # this will put the cluster in retry-wait when get_clusters runs
 
+    def set_build_image(self):
+        """
+        Overrides build_image for worker, to be same as in orchestrator build
+        """
+        spec = get_build_json().get("spec", {})
+        self.config_kwargs['build_image'] = spec['strategy']['customStrategy']['from']['name']
```

Comment:
This seems to work as expected, but let's make this easier to debug if something goes wrong.
Verify 'kind' is 'DockerImage'. If it's not, it means OpenShift didn't convert the ImageStreamTag into a docker image reference as we expect. In this case, let's raise an error as I don't think there's a sane way to recover from it.

Reply:
`.get()` should be used to get items of other arrays as well, so that a nicer exception could be thrown

Reply:
.get() will not throw anything, or are you suggesting to, use get on every key, and check if it isn't None?

Reply:
will do check for 'DockerImage'

Reply:
Since there doesn't seem to be a nice way of chaining `.get()` calls, lets wrap this function in `try..catch KeyError` and throw `RuntimeError` with a better description - e.g. `Build object is malformed, failed to fetch buildroot image`

Reply:
This hasn't been addressed yet.

Reply:
it will be